#goal
LabelToFeatures[] elements
---
#result
bool training_success
---
#feedback
float32 percent_complete
